src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.cs

```diff
@@ -16,7 +16,12 @@ namespace Microsoft.CodeAnalysis.SQLite
     /// </summary>
     internal partial class SQLitePersistentStorage : AbstractPersistentStorage
     {
-        private const string Version = "1";
+        // Version history.
+        // 1. Initial use of sqlite as the persistence layer.  Simple key->value storage tables.
+        // 2. Updated to store checksums.  Tables now key->(checksum,value).  Allows for reading
+        //    and validating checksums without the overhead of reading the full 'value' into
+        //    memory.
+        private const string Version = "2";
```

Comment:
note: version changes in SQLitePersistentStorage  are done in a forward/backward compatible manner.  A single DB can be shared between vs2017 and vs2019 without issues.

Reply:
> A single DB can be shared between vs2017 and vs2019 without issues.

This is not true until we have regression tests proving it to be true. We need a mechanism to ensure a database created by VS2017 is not loaded in VS2019.

Reply:
I'm happy to write a regression test for this